ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -28,76 +32,167 @@ const model: Model = {
                 height: 75,
+                entityType: 'DEPLOYMENT',
+            },
+        },
+        {
+            id: '__MzQuMTIwLjAuMC8xNg',
+            label: 'Google/global | 34.120.0.0/16',
+            type: 'node',
+            width: 75,
+            height: 75,
+            data: {
+                id: '__MzQuMTIwLjAuMC8xNg',
+                label: 'Google/global | 34.120.0.0/16',
+                type: 'node',
+                width: 75,
+                height: 75,
+                entityType: 'CIDR_BLOCK',
+            },
+        },
+        {
+            id: 'afa12424-bde3-4313-b810-bb463cbe8f90',
+            label: 'External entities',
+            type: 'node',
+            width: 75,
+            height: 75,
+            data: {
+                id: 'afa12424-bde3-4313-b810-bb463cbe8f90',
+                label: 'External entities',
+                type: 'node',
+                width: 75,
+                height: 75,
+                entityType: 'EXTERNAL_ENTITIES',
             },
         },
         {
-            id: 'group1',
+            id: 'e8dabcb7-f471-414e-a999-fe91be5a28fa',
             type: 'group',
-            children: ['n1', 'n2'],
+            children: [
+                'e337f873-64d8-46be-84ed-2a0c38c75fac',
+                '09134b5d-8c12-41e8-821b-c97a5a1331c9',
+            ],
             group: true,
             label: 'stackrox',
             style: { padding: 15 },
             data: {
                 collapsible: true,
                 showContextMenu: false,
+                entityType: 'NAMESPACE',
+            },
+        },
+        {
+            id: 'EXTERNAL',
+            type: 'group',
+            children: ['__MzQuMTIwLjAuMC8xNg', 'afa12424-bde3-4313-b810-bb463cbe8f90'],
+            group: true,
+            label: 'External to cluster',
+            style: { padding: 15 },
+            data: {
+                collapsible: true,
+                showContextMenu: false,
+                entityType: 'EXTERNAL',
             },
         },
     ],
     edges: [
         {
             id: 'e1',
             type: 'edge',
-            source: 'n1',
-            target: 'n2',
+            source: 'e337f873-64d8-46be-84ed-2a0c38c75fac',
+            target: '09134b5d-8c12-41e8-821b-c97a5a1331c9',
         },
         {
             id: 'e2',
             type: 'edge',
-            source: 'n2',
-            target: 'n1',
+            source: '09134b5d-8c12-41e8-821b-c97a5a1331c9',
+            target: 'e337f873-64d8-46be-84ed-2a0c38c75fac',
+        },
+        {
+            id: 'e3',
+            type: 'edge',
+            source: '__MzQuMTIwLjAuMC8xNg',
+            target: 'e337f873-64d8-46be-84ed-2a0c38c75fac',
+        },
+        {
+            id: 'e4',
+            type: 'edge',
+            source: 'afa12424-bde3-4313-b810-bb463cbe8f90',
+            target: 'e337f873-64d8-46be-84ed-2a0c38c75fac',
         },
     ],
 };
 
+function findEntityById(graphModel, id) {
```

Comment:
What do you think about defining these functions in separate modules? That way we can also create a test file for each of them

Reply:
I would imagine we'll eventually have different types of helper functions for the graph:
* Finding nodes/edges using some condition (id, name, namespace, etc.)
* Filtering nodes/edges by some condition (connected to a specific node, etc.)

Reply:
Will definitely pull these out into a separate file. I just started with them here while I was exploring options.